css/style.css

```diff
@@ -261,6 +251,68 @@ header h2 {
   align-items: center;
 }
 
+.mobile-menu-hide {
+  display: none;
+}
+
+.mobile-menu-show {
+  position: absolute;
+  top: 0%;
+  right: 0%;
+  left: 0%;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  background-color: var(--secondary-color);
+  color: #fff;
+  border: 1px #2a9df4 solid;
+  border-radius: 20px;
+}
+
+.mobile-menu-show > ul > li {
+  margin-left: 28px;
+  margin-top: 40px;
+  font-size: 24px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+.mobile-menu-show > ul {
+  display: block;
+  list-style-type: disc;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  padding-inline-start: 40px;
+}
+
+.mobile-menu-show > ul > li > a {
+  font-size: 24px;
+  font-weight: 600;
+  color: var(--primary-color);
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.mobile-menu-show > button {
+  margin: 0.5em 0.5em;
+  align-self: flex-end;
+  cursor: pointer;
+  background-color: var(--secondary-color);
+}
+
+.mobile-menu-show > i {
+  width: 2.5em;
+}
+
+.btn-close-nav {
```

Comment:
- [ ] The close button has a ```grey``` color, kindly change it to ```#67798E``` (the color could be different because of the opacity of the background)